Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift

```diff
@@ -729,6 +730,14 @@ public struct RenderNodeTranslator: SemanticVisitor {
             }
         }
 
+        if let customMetadatas = documentationNode.metadata?.customMetadata {
+            node.metadata.customMetadata = customMetadatas.map { customMetadata -> CustomMetadataDictionary in
+                let key = customMetadata.key
+                let value = customMetadata.value
+                return CustomMetadataDictionary(key: key, value: value!)
```

Comment:
Why are we force unwrapping `value` here? Can we adjust the model so that this value can't be nil?

Reply:
this has been updated to be not nil